core/src/test/java/org/elasticsearch/indices/IndicesServiceTests.java

```diff
@@ -187,4 +199,54 @@ public void testPendingTasks() throws Exception {
         assertAcked(client().admin().indices().prepareOpen("test"));
 
     }
+
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/17695")
+    public void testDeletingSameIndexDirectoryFromConcurrentProcesses() throws Exception {
```

Comment:
I don't think we should keep this test. I dont' think it adds much value as is now (but I understand why you had it when trying things before!)